### ISSUE TITLE
Cannot download any file from Chromium browser

### ISSUE THREAD

**Issue by Carl393** [dnschneid/crouton#3927]
```
name: buster
encrypted: no
Entering /home/linux/chroots/buster...
crouton: version 1-20181001133934~master:38012fdf
release: buster
architecture: amd64
xmethod: xorg
targets: xfce
host: version 11151.17.0 (Official Build) beta-channel auron_yuna 
kernel: Linux localhost 3.14.0 #1 SMP PREEMPT Tue Oct 30 00:45:31 PDT 2018 x86_64 GNU/Linux
freon: yes
```

#### Please describe your issue:
every time I try to save a file to the system it fails...

#### If known, describe the steps to reproduce the issue:
downloading anything from chromium using xfce.

**Comment by DennisLfromGA** [comment]
@Carl393,

Where are you trying to save the file(s), in ~/Downloads?
Can you save a file in Chrome OS but not in your crouton chroot?

-**DennisLfromGA**

**Comment by Carl393** [comment]
Yes Sorry for the delay.. when in xfce4 i cannot save a file in the shared downloads folder... under /home/carl/downloads/ but I can save a file on the xfce desktop.... is this a permissions problem? I never had this issue before.

**Comment by Carl393** [comment]
![screenshot_2018-11-09_18-08-39](https://user-images.githubusercontent.com/14049856/48292942-9a2aa900-e44a-11e8-967f-a5968ced6d77.png)

I get this message... any clue what it means?

**Comment by Carl393** [comment]
It seems like it belongs to Crouton?
How do i make it also belong to me? (username carl)
I find this kind of funny I realize it's a shared folder but some programs like SKYPE pick this folder by default. it would be nice if it just worked by default.

I can save any file to this folder using chrome OS but not when I'm in crouton.

![Screenshot_2020-05-18_05-56-19](https://user-images.githubusercontent.com/14049856/82199914-541a9500-98cc-11ea-9029-7b05be9bea07.png)

**Comment by DennisLfromGA** [comment]
@Carl393,

I don't believe the ownership of the ~/Downloads directory is the issue, mine are shown below and I have no problem when saving to ~/Downolads in the chroot -
```
(xenial)denny@localhost:~$ ll
total 24K
drwxr-xr-x.  3 denny denny   4.0K Dec 26 16:22 bin/
drwxrwxr-x.  2 denny denny   4.0K Oct 13  2016 Desktop/
drwx--x---. 15 denny crouton  12K May 18 09:04 Downloads/

(xenial)denny@localhost:~$ mount| grep /home
/dev/sda1 on /home/denny/Downloads type ext4 (rw,nosuid,nodev,noatime,seclabel,resgid=20119,commit=600,data=ordered)
```
I can't reproduce the issue myself but you may find some help or clues here as it appears to be an upstream bug in 'buster':

> Cannot create files in `~/Downloads` · Issue #3860 · dnschneid/crouton
> https://github.com/dnschneid/crouton/issues/3860

> Required key not available · Issue #3261 · dnschneid/crouton
> https://github.com/dnschneid/crouton/issues/3261

*Hope this helps,*
-**DennisLfromGA**
